lib/structures/Message.js

```diff
@@ -108,7 +80,7 @@ class Message extends Base {
 
     update(data, client) {
         if(this.type === 3) { // (╯°□°）╯︵ ┻━┻
-            (this.channel.call || this.channel.lastCall).update(data.call);
+            this.channel.call.update(data.call);
```

Comment:
as noted above, `this.channel.call` is no longer a thing

Reply:
actually, messages will never have type 3 anymore anyway, just remove this condition entirely